### ISSUE TITLE
Install error with pip: Ignored the following yanked versions

### ISSUE THREAD

**Issue by salamanders** [google/yggdrasil-decision-forests#123]
Getting a strange error with a brand new venv:

    (myenv24b) myusername-macbookpro3:myenv24b myusername$ pip install ydf -U
    ERROR: Ignored the following yanked versions: 0.0.0
    ERROR: Could not find a version that satisfies the requirement ydf (from versions: none)
    ERROR: No matching distribution found for ydf

**Comment by Spider-netizen** [comment]
I'm getting the same error messages.

**Comment by rstz** [comment]
Release 0.0.0 did not include any code, it's just the shell of a Python package. Unsurprisingly, such a package is compatible with every possible architecture, so we decided to yank the package. If you're seeing this message, it probably means you are using an operating system / architecture that is currently not supported

Our team currently publishes and supports Linux x64 and Windows releases. Unfortunately, we do not have the bandwidth, at this time, to support other architectures. I'm happy to help you with compiling yourself - we've experimented with a number of architectures and things mostly worked (no promises!)
